server/src/test/java/org/elasticsearch/cluster/routing/allocation/AllocationCommandsTests.java

```diff
@@ -245,6 +251,29 @@ public void testAllocateCommand() {
             fail("expected IllegalArgumentException when allocating shard while no unassigned shard available");
         } catch (IllegalArgumentException e) {
         }
+
+        // mark all shards as stale
```

Comment:
any chance we can extract it to it's own test method? no need to make this test even longer.

Reply:
addressed it e14d094